<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>662</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Cadastro Produto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 28, 61);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>9</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="4" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Pagamento:</string>
        </property>
       </widget>
      </item>
      <item row="22" column="2" colspan="2">
       <widget class="QPushButton" name="save">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 218, 218);
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Salvar Alteração</string>
        </property>
       </widget>
      </item>
      <item row="21" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_9">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>CNPJ:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="Pesquisar">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 218, 218);</string>
        </property>
        <property name="text">
         <string>Pesquisar</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="22" column="0">
       <widget class="QPushButton" name="cadastro">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 218, 218);
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Adicionar </string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Valor Unitário:</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Data da Compra:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLineEdit" name="lineEdit_5">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Para alterar um produto, digite seu código.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="22" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Quantidade:</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Descrição:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Valor total:</string>
        </property>
       </widget>
      </item>
      <item row="14" column="2">
       <widget class="QPushButton" name="calcula">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(218, 218, 218);
border-radius: 10px;</string>
        </property>
        <property name="text">
         <string>Calcula</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Cadastrar uma compra</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_6">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="2">
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>Pesquisar</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>calcula</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>cadastro</tabstop>
  <tabstop>save</tabstop>
  <tabstop>lineEdit_7</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>save</sender>
   <signal>released()</signal>
   <receiver>lineEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>released()</signal>
   <receiver>lineEdit_3</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>released()</signal>
   <receiver>lineEdit_4</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cadastro</sender>
   <signal>released()</signal>
   <receiver>lineEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cadastro</sender>
   <signal>released()</signal>
   <receiver>lineEdit_3</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cadastro</sender>
   <signal>released()</signal>
   <receiver>lineEdit_4</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>released()</signal>
   <receiver>lineEdit_5</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>681</x>
     <y>623</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cadastro</sender>
   <signal>released()</signal>
   <receiver>lineEdit_6</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>606</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cadastro</sender>
   <signal>released()</signal>
   <receiver>lineEdit_7</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>601</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cadastro</sender>
   <signal>released()</signal>
   <receiver>lineEdit_8</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>149</x>
     <y>600</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cadastro</sender>
   <signal>released()</signal>
   <receiver>lineEdit_9</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>released()</signal>
   <receiver>lineEdit_6</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>530</x>
     <y>589</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>released()</signal>
   <receiver>lineEdit_7</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>580</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>released()</signal>
   <receiver>lineEdit_8</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>562</x>
     <y>594</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>502</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>released()</signal>
   <receiver>lineEdit_9</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>595</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>568</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
